Library/Homebrew/os/linux/elf.rb

```diff
@@ -68,10 +68,24 @@ def binary_executable?
     elf_type == :executable
   end
 
+  def with_interpreter?
```

Comment:
I suggest renaming this function to `has_interpreter?`.

Reply:
`has_interpreter?` causes rubocop complaining `Rename has_interpreter? to interpreter?`.

So I use `with_interpreter?` instead.

Reply:
@MikeMcQuaid Any opinion/preference?

Reply:
`interpreter?` or `with_interpreter?` both work for me (with slight preference to the prior).

Reply:
I don't like `interpreter?` as it feels like `is_interpreter?` rather than `has_interpreter?`.